tests/client.go

```diff
@@ -0,0 +1,64 @@
+package test
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+func Request(
+	baseUrl, 
+	method string, 
+	inputBody io.Reader, 
+	headers map[string]string,
+	uris ...string) (*http.Response, map[string]interface{}, error) {
+
+	url := baseUrl
+	for _, uri := range uris {
+		url = fmt.Sprintf("%s%s", url, uri)
+	}
+
+	client := &http.Client{}
+
+	req, err := http.NewRequest(method, url, inputBody)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	if len(headers) > 0 {
```

Comment:
This condition seems unnecessary since `range` of an empty map would do nothing.

Reply:
this is true, but maybe a nil check is neseccary?

Reply:
Yeah, that can be good